spec/features/invite_and_register_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+feature 'A new user registers for myflix', :js, :vcr do
+  given!(:riggs) { Fabricate :user, full_name: 'Riggs', email: 'riggs@example.com', password: 'password' }
+  given!(:invite_attributes) { Fabricate.attributes_for :invitation }
+
+  context 'from an invitation email' do
+    background do
+      login riggs
+      visit new_invitation_path
+    end
+
+    scenario 'user invites a friend with an unregistered email' do
+      invite_friend_via_email
+      accept_email_invite
+      provide_user_information
+      subscribe_with_credit_card '4242424242424242'
+      expect_successful_registration
+
+      invitee_signs_in
+      user_should_follow name_string: riggs.full_name
```

Comment:
`_string` doesn't really follow the duck typing principle. I would just name it `name`.